card/lib/card/mod/loader.rb

```diff
@@ -89,6 +90,13 @@ def source_files card
           files.flatten
         end
 
+        def load_initializers
+          Cardio.config.paths["mod/config/initializers"].existent
```

Comment:
Looks like in this file we are using Card.paths. In other places we are using Cardio.paths. What's the difference?